app/parsers/bulkrax/oai_dc_parser.rb

```diff
@@ -1,6 +1,8 @@
 module Bulkrax
   class OaiDcParser < ApplicationParser
-    attr_accessor :client, :headers
+    include Hyrax::Lockable
```

Comment:
you include a lock module here, but not in csv_parser. Is there a missing include? Should this include go up to application_parser?